ivy/functional/ivy/nest.py

```diff
@@ -711,6 +711,32 @@ def copy_nest(
     ret
         The copied nest.
 
+    Examples
+    --------
+    With :code:`ivy.Array` input:
+
+    >>> nest = ivy.array([1.,2.,3.],[7.,8.,9.])
```

Comment:
Please update this to `nest = ivy.array([[1.,2.,3.],[7.,8.,9.]])`

Reply:
Hi, I handled this problem by updating `ivy.array`.